docs/configure-custom-values-envoy.md

```diff
@@ -169,3 +157,18 @@ You can specify the number of replicas (pods) of Scalar Envoy using `envoy.repli
 envoy:
   replicaCount: 3
 ```
+
+### Taints/Tolerations configurations (Optional based on your environment)
+
+If you want to control pod deployment using the taints and tolerations of Kubernetes, you can use `envoy.tolerations`.
+
+You can configure them using the same syntax as the tolerations of Kubernetes. So, please refer to the official document [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more details on the tolerations configuration of Kubernetes.
+
+```yaml
+envoy:
+  tolerations:
+    - effect: NoSchedule
+      key: scalar-labs.com/dedicated-node
+      operator: Equal
+      value: scalardb
+```
```

Comment:
I moved this section from `Recommended in the production environment` to `Optional based on your environment`.
This is because we don't recommend too strict constraints in the new recommendations.

However, users can set this configuration if they want based on their system's requirements.